scripts/mirador.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+echo "Installing Mirador"
+
+HOME_DIR=$1
+
+if [ -f "$HOME_DIR/islandora/configs/variables" ]; then
+  . "$HOME_DIR"/islandora/configs/variables
+fi
+
+if [ ! -f "$DOWNLOAD_DIR/master.zip" ]; then
+    echo "Downloading mirador-js dependencies"
+    cd "$DOWNLOAD_DIR" && { curl -LOk https://github.com/NVLI/mirador-js/archive/master.zip; cd -;}
+    cd "$DRUPAL_HOME"/web/libraries && { unzip "$DOWNLOAD_DIR"/master.zip -d . ;}
```

Comment:
@flummingbird this line needs to move outside the `if...fi` (line 16) as it should occur whenever a box is provisioned and is separate from the downloading. 
The way I think of it is that lines 10-12, is just checking if you have downloading the application before. 
Then line 13 is deploying that application into your vagrant machine.

So you can conceivably run lines 10-12 once and use line 13 waaaaaay more than that.